Mods/ArchipelagoCore.SC2Mod/Base.SC2Data/Lib5BD4895D_h.galaxy

```diff
@@ -127,24 +129,27 @@ void lib5BD4895D_gf_AP_Core_setFactionColor (string lp_faction, int lp_colorInde
 void lib5BD4895D_gf_AP_Core_onLoadFinished ();
 void lib5BD4895D_gf_AP_Core_SwitchKerriganSwarmToPrimalZerg ();
 void lib5BD4895D_gf_AP_Core_initObjectivePanel ();
-void lib5BD4895D_gf_AP_Core_addObjective (int lp_objective, int lp_category, int lp_checks, bool lp_hasCounter, string lp_counterSuffix, int lp_counterValue, int lp_counterMax, bool lp_hasTimer, timer lp_timer);
+void lib5BD4895D_gf_AP_Core_addObjective (int lp_objective, int lp_category, bool lp_hasCounter, string lp_counterSuffix, int lp_counterValue, int lp_counterMax, bool lp_hasTimer, timer lp_timer, bool lp_hideIfNoLocs);

```

Comment:
(note for other reviewers) Confirmed this function is only called within AP_Core mod, so this doesn't break an interface we're actually using in any maps. The `addCategoryObjective*()` functions changed below are only called by maps using this new scheme, which have all been updated by this PR.

Reply:
Correct. This is not directly used in any maps, so this will not break existing content.